examples/rich_versions.py

```diff
@@ -0,0 +1,21 @@
+from uvtrick import Env
+
+rich_versions = (10, 11, 12, 13)
+print(f"Now iterating through {rich_versions=}:")
+
+
+def uses_rich(a, b):
+    from rich import print
+    from importlib import metadata
+
+    version = metadata.version("rich")
+
+    print(f"hello from rich=={version}")
+    return a + b
```

Comment:
This is a nice touch to make the example more clear! Feel free to update the example on the README.md with this one.

Reply:
Contributed in 46e130d !